packages/plans-grid-next/src/components/comparison-grid/index.tsx

```diff
@@ -662,7 +662,7 @@ const ComparisonGridFeatureGroupRowCell: React.FunctionComponent< {
 									{ feature.getCompareSubtitle() }
 								</span>
 							) }
-							{ hasFeature && ! featureLabel && <Gridicon icon="checkmark" color="#0675C4" /> }
+							{ hasFeature && ! featureLabel && <Gridicon icon="checkmark" color="#3858e9" /> }
```

Comment:
Non-blocking note: by a quick grep, this color code has been used extensively across the codebase. We should consider to standardize it.

Reply:
Good idea, updated to `var(--studio-wordpress-blue-50)`.